packages/cashier/src/stores/on-ramp-store.ts

```diff
@@ -219,8 +230,8 @@ export default class OnRampStore extends BaseStore {
     @action.bound
     resetPopup() {
         this.setApiError(null);
-        this.setDepositAddress(null);
-        this.setDepositAddressRef(null);
+        this.setDepositAddress('');
+        this.setDepositAddressRef({} as Node);
```

Comment:
just a query why Node ?

Reply:
@vinu-deriv It's node ref here so It should be Node.